### PR TITLE
Add Redis monitoring check at Step 5

### PR DESCRIPTION
**Pull request by dangiankit**
Demonstrates the use of docker-compose exec by adding a Redis monitoring check at Step 5 of the microservice orchestration tutorial to understand that the enhanced requests were served from the Redis cache.

### CODE REVIEW

File: _posts/2018-09-22-microservice-orchestration.markdown [hunk 1]
```diff
@@ -866,7 +866,11 @@ Creating linkextractor_redis_1 ... done
 
 Now, that all three services are up, access the web interface by [clicking the Link Extractor](/){:data-term=".term1"}{:data-port="80"}.
 There should be no visual difference from the previous step.
-However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly.
+However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly. To check whether or not the Redis service is being used, we use `docker-compose exec` followed by the `redis` service name and the Redis CLI's [Monitor](https://redis.io/commands/monitor) command:
```
Comment (ibnesayeed) [inline _posts/2018-09-22-microservice-orchestration.markdown]:
```suggestion
However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly.
To check whether or not the Redis service is being utilized, we can use `docker-compose exec` followed by the `redis` service name and the Redis CLI's [monitor](https://redis.io/commands/monitor) command:
```

I have followed one line per sentence convention when writing this Markdown file for better diffs. The above inline suggestion takes care of that and some minor changes in the sentence. You can either [incorporate these suggestions](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request) or make necessary changes yourself.

**Review by ibnesayeed — CHANGES_REQUESTED** [review]

**Review by ibnesayeed — CHANGES_REQUESTED** [review]
A side-effect of this command is that the terminal is now in the interactive mode and followup commands will not work until the process is killed or sent to background. There should be clear instructions to come out of that mode (e.g., `CTRL + C`) or better yet a click to run-style terminal command (if possible).

File: _posts/2018-09-22-microservice-orchestration.markdown [hunk 2]
```diff
@@ -867,6 +867,13 @@ Creating linkextractor_redis_1 ... done
 Now, that all three services are up, access the web interface by [clicking the Link Extractor](/){:data-term=".term1"}{:data-port="80"}.
 There should be no visual difference from the previous step.
 However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly.
+To check whether or not the Redis service is being utilized, we can use `docker-compose exec` followed by the `redis` service name and the Redis CLI's [monitor](https://redis.io/commands/monitor) command:
+
+```.term1
+docker-compose exec redis redis-cli monitor
+```
+
+To leave the interactive shell and continue with the rest of the tutorial, use SIGINT (Ctrl-C) to stop the `monitor` stream running via `redis-cli`.
```
Comment (ibnesayeed) [inline _posts/2018-09-22-microservice-orchestration.markdown]:
```suggestion
Now, try to extract links from some web pages using the web interface and see the difference in Redis log entries for pages that are scraped the first time and those that are repeated.
Before continuing further with the tutorial, stop the interactive `monitor` stream as a result of the above `redis-cli` command by pressing `Ctrl + C` keys while the interactive terminal is in focus.
```

Trying to simplify the instructions for beginners who might not know much about signals.

**Review by ibnesayeed — CHANGES_REQUESTED** [review]
Some more clarification and rephrasing is suggested.

**Review by ibnesayeed — APPROVED** [review]
LGTM!

Thanks for this PR @dangiankit.
@marcosnils or someone from the PWD maintainers team should be able to merge it.

**Comment by marcosnils** [comment]
Thanks everyone for taking the time to improve the lab!.

**Pull request merged**